cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRunnerClassLoader.java

```diff
@@ -63,8 +63,9 @@
     Type.getObjectType("org/apache/spark/streaming/StreamingContext");
   private static final Type SPARK_CONF_TYPE = Type.getObjectType("org/apache/spark/SparkConf");
   // SparkSubmit is a companion object, hence the "$" at the end
-  private static final Type SPARK_SUBMIT_TYPE = Type.getObjectType("org.apache.spark.deploy.SparkSubmit$");
```

Comment:
Because we just use the getClassName, which do a replace of / to .
However, since it is not right to have an internal name with ., hence fixing it